checkr/src/middlewares/request-validator-middleware.ts

```diff
@@ -0,0 +1,13 @@
+import { NextFunction, Request, Response } from "express";
+import { z } from "zod";
+
+export const validateRequest = (schema: z.ZodObject<any, any>) => {
+    return (req: Request, res: Response, next: NextFunction) => {
+        try {
+            schema.parse(req.body);
+            next();
+        } catch (error: any) {
+            res.status(400).json({ error: error.message });
```

Comment:
here you forgot to add httpStatus library

Reply:
Will correct it.